internal/services/eventhub/eventhub_namespace_resource.go

```diff
@@ -190,6 +190,29 @@ func resourceEventHubNamespace() *pluginsdk.Resource {
 				},
 			},
 
+			"local_authentication_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
is this also the default state the API returns when provisioning on an older version and upgrading to a newer api version?

Reply:
yes, I tried the older version and the default value is the same